orc8r/lib/go/service/config/service_config.go

```diff
@@ -33,9 +33,17 @@ var (
 	configDir         = "/etc/magma/configs"
 	oldConfigDir      = "/etc/magma"
 	configOverrideDir = "/var/opt/magma/configs/"
+	specDir           = "/etc/magma/configs/orc8r/specs/"
 	cfgDirMu          sync.RWMutex
 )
 
+// GetSpecPath returns the file on the production image
+// that contains the service's Swagger spec
+func GetSpecPath(service string) string {
```

Comment:
Move this function to the end of the public functions list -- it's less important / not the principal focus of this file